plugin-server/src/utils/db/hub.ts

```diff
@@ -97,6 +98,19 @@ export async function createHub(
     })
     logger.info('👍', `ClickHouse ready`)
 
+    logger.info('🤔', `Connecting to Cassandra...`)
+    const cassandra = new CassandraClient({
```

Comment:
I don't think this should be on the hub. The hub is used by the entire codebase where its literally only this one service that needs it. 

If it didn't call `connect` it would be less of a big deal but it does so lets move it to the dedicated consumer for now